Mathlib/Analysis/SpecialFunctions/ImproperIntegrals.lean

```diff
@@ -177,3 +177,31 @@ theorem integral_Ioi_cpow_of_lt {a : ℂ} (ha : a.re < -1) {c : ℝ} (hc : 0 < c
   simp_rw [neg_neg, Complex.norm_eq_abs, Complex.abs_cpow_eq_rpow_re_of_pos hx, Complex.add_re,
     Complex.one_re]
 #align integral_Ioi_cpow_of_lt integral_Ioi_cpow_of_lt
+
+private theorem intervalIntegral_one_div_one_add_sq_tendsto :
+    Tendsto (fun i => ∫ (x : ℝ) in -i..i, 1 / (1 + x ^ 2)) atTop (𝓝 π) := by
```

Comment:
I think you should prefer `(1 + x ^ 2)⁻¹` throughout this PR since that is the simp normal form. Of course, please rename the lemmas appropriately too.

Also, it's a minor point but you can strengthen this lemma to `𝓝[<] π`.